model/utils.go

```diff
@@ -52,10 +58,14 @@ func (er *AppError) Error() string {
 }
 
 func (er *AppError) Translate(T goi18n.TranslateFunc) {
-	if er.params == nil {
-		er.Message = T(er.Id)
+	if T != nil {
+		if er.params == nil {
+			er.Message = T(er.Id)
+		} else {
+			er.Message = T(er.Id, er.params)
+		}
 	} else {
-		er.Message = T(er.Id, er.params)
+		er.Message = er.Id
```

Comment:
0/5 but this could be written a bit cleaner like this:

```Go
if T == nil {
  er.Message = er.Id
  return
}

if er.params == nil {
  er.Message = T(er.Id)
} else {
  er.Message = T(er.Id, er.params)
}

Reply:
Done